dl/src/main/scala/com/intel/analytics/bigdl/dataset/Transformer.scala

```diff
@@ -116,19 +120,40 @@ class SampleToBatch[T: ClassTag]
             if (labelData == null || labelData.length < batchSize * labelLength) {
               labelData = new Array[T](batchSize * labelLength)
             }
-            sample.copyToLabel(labelData, i*labelLength, labelLength)
-            sample.copyToFeature(featureData, i*featureLength, featureLength)
+            sample.copyLabel(labelData, i*labelLength, labelLength)
+            sample.copyFeature(featureData, i*featureLength, featureLength)
 
+            featureTable(i + 1) = Tensor()
```

Comment:
Shall we create an array of Tensor, the size of array set to batchSize, to reuse?